sdk/typescript/runtime/main.go

```diff
@@ -177,38 +186,70 @@ func (t *TypeScriptSdk) CodegenBase(ctx context.Context, modSource *ModuleSource
 		},
 	})
 
+	moduleFiles, err := base.Directory(".").Entries(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("could not list rootfs entries: %v", err)
+	}
+	packageJsonExist := slices.Contains(moduleFiles, "package.json")
+
 	switch detectedRuntime {
 	case Bun:
-		base = base.
-			// Check if the project has existing source:
-			// if it does: add sdk as dev dependency
-			// if not: copy the template and replace QuickStart with the module name
-			WithExec([]string{"sh", "-c",
-				"if [ -f package.json ]; then  bun install ./sdk  --dev  && bun /opt/module/bin/__tsconfig.updator.ts; else cp -r /opt/module/template/*.json .; fi",
-			})
+		// Check if the project has existing source:
+		// if it does: add sdk as dev dependency
+		// if not: copy the template and replace QuickStart with the module name
+		if packageJsonExist {
+			base = base.
+				WithExec([]string{"bun", "install", genDir}).
+				WithExec([]string{"bun", "/opt/module/bin/__tsconfig.updator.ts"})
+		} else {
+			base = base.
+				WithExec([]string{"sh", "-c", "cp -r /opt/module/template/*.json ."})
```

Comment:
Can this be done using a `WithDirectory` and an `Include` filter? (also same below)

Reply:
Will it merge the current `.` with what I provide? If yes then I can

Reply:
Yep it'll merge (I forget that all the time)